app/javascript/angular/code/controllers/_info_window_ctrl.js

```diff
@@ -1,8 +1,31 @@
-export const InfoWindowCtrl = ($scope, sensors, infoWindow, map) => {
+export const InfoWindowCtrl = ($scope, sensors, infoWindow, map, heat) => {
   $scope.sensors = sensors;
   $scope.infoWindow = infoWindow;
 
   $scope.zoomIn = () => {
     map.zoomToSelectedCluster();
   };
+
+  $scope.class = () =>
+    "info_window__avg-color " +
+    classByValue(heat.getLevel(Math.round(infoWindow.data.average)));
+};
+
+const classByValue = value => {
```

Comment:
What do you think of keeping this in the `heat` module?

Reply:
There could be something like `heat.classByValue(infoWindow.data.average)`

Reply:
Good point! thanks